### PR TITLE
Post upgrade hook fix

### PR DESCRIPTION
**Pull request by nighthawq7**
#### What this PR does / why we need it:
It allows users of this helm chart to opt-out for the post upgrade hook.

#### Which issue this PR fixes
  - fixes 341

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by Jasstkn — APPROVED** [review]
Thanks for the contribution! LGTM. 🚀

**Comment by Jasstkn** [comment]
@imrajdas @ispeakc0de hey folks. Can I ask you to take a look?

**Comment by Jasstkn** [comment]
> @imrajdas @ispeakc0de hey folks. Can I ask you to take a look?

ping

**Comment by nighthawq7** [comment]
Not sure where to ask, but I see that the chart version is of 3.0.1 on master, but that version have not been released. 
When/What triggers a release of the charts?

**Comment by Jasstkn** [comment]
> Not sure where to ask, but I see that the chart version is of 3.0.1 on master, but that version have not been released. When/What triggers a release of the charts?

thanks for the report. fixed: https://github.com/litmuschaos/litmus-helm/releases/tag/litmus-3.0.1

File: charts/litmus/values.yaml [hunk 1]
```diff
@@ -57,6 +57,7 @@ ingress:
   #    hosts: []
 
 upgradeAgent:
+  enabled: false
```
Comment (gdsoumya) [inline charts/litmus/values.yaml:60]: The default should be enabled

**Review by gdsoumya — CHANGES_REQUESTED** [review]

**Comment by nighthawq7** [comment]
@Jasstkn @imrajdas @ispeakc0de 
I'm fairly new to contributing to an open source project. What is the normal lead time to get a pull request through?
To me it seems like a very slow pace at the moment and that will get potential consumer to hesitate to use the tool.

**Review by gdsoumya — APPROVED** [review]
LGTM

**Comment by gdsoumya** [comment]
@nighthawq7 Thanks for the contribution!

**Pull request merged**